clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -2041,7 +2041,7 @@ void CIRGenModule::emitTopLevelDecl(Decl *decl) {
     case PCK_Unknown:
       llvm_unreachable("unexpected pragma comment kind");
     case PCK_Linker:
-      assert(!MissingFeatures::emitModuleLinkOptions() && "NYI");
+      appendLinkerOptions(PCD->getArg());
```

Comment:
If you can't stress this path because we currently lack support, you shouldn't change anything under `lib/CIR/CodeGen` as part of this PR. Alternatively you can add the missing windows support in another PR and re-introduce this later.